app-misc/rpick/rpick-0.4.0.ebuild

```diff
@@ -0,0 +1,91 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# Auto-Generated by cargo-ebuild 0.2.0
+
+EAPI=7
+
+CRATES="
+ansi_term-0.11.0
+arrayref-0.3.5
+arrayvec-0.4.12
+atty-0.2.13
+autocfg-0.1.7
+backtrace-0.3.40
+backtrace-sys-0.1.32
+base64-0.10.1
+bitflags-1.2.1
+blake2b_simd-0.5.8
+byteorder-1.3.2
+cc-1.0.46
+cfg-if-0.1.10
+clap-2.33.0
+cloudabi-0.0.3
+constant_time_eq-0.1.4
+crossbeam-utils-0.6.6
+dirs-1.0.5
+dtoa-0.4.4
+failure-0.1.6
+failure_derive-0.1.6
+fuchsia-cprng-0.1.1
+heck-0.3.1
+lazy_static-1.4.0
+libc-0.2.65
+linked-hash-map-0.5.2
+nodrop-0.1.14
+proc-macro2-0.4.30
+proc-macro2-1.0.6
+quote-0.6.13
+quote-1.0.2
+rand-0.6.5
+rand_chacha-0.1.1
+rand_core-0.3.1
+rand_core-0.4.2
+rand_hc-0.1.0
+rand_isaac-0.1.1
+rand_jitter-0.1.4
+rand_os-0.1.3
+rand_pcg-0.1.2
+rand_xorshift-0.1.1
+rdrand-0.4.0
+redox_syscall-0.1.56
+redox_users-0.3.1
+rpick-0.4.0
+rust-argon2-0.5.1
+rustc-demangle-0.1.16
+serde-1.0.102
+serde_derive-1.0.102
+serde_yaml-0.8.11
+strsim-0.8.0
+structopt-0.2.18
+structopt-derive-0.2.18
+syn-0.15.44
+syn-1.0.7
+synstructure-0.12.1
+textwrap-0.11.0
+unicode-segmentation-1.5.0
+unicode-width-0.1.6
+unicode-xid-0.1.0
+unicode-xid-0.2.0
+vec_map-0.8.1
+winapi-0.3.8
+winapi-i686-pc-windows-gnu-0.4.0
+winapi-x86_64-pc-windows-gnu-0.4.0
+yaml-rust-0.4.3
+"
+
+inherit cargo
+
+DESCRIPTION="Helps you pick items from a list by various algorithms"
+HOMEPAGE="https://gitlab.com/bowlofeggs/rpick"
+SRC_URI="$(cargo_crate_uris ${CRATES})"
+LICENSE="GPL-3 Apache-2.0 BSD-2 CC0-1.0 ISC MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+DOCS="CHANGELOG.md README.md"
```

Comment:
You can remove this, as `einstalldocs` should already match these filenames. In future, this should also be in array form (since it gets **converted** to array form if it's not).

Reply:
I added this because ```CHANGELOG.md``` wasn't getting installed.

Reply:
I just double checked, and without this this happens:

```
$ equery f rpick
 * Searching for rpick ...
 * Contents of app-misc/rpick-0.4.0:
/usr
/usr/bin
/usr/bin/rpick
/usr/share
/usr/share/doc
/usr/share/doc/rpick-0.4.0
/usr/share/doc/rpick-0.4.0/README.md.bz2
```

Reply:
Hmm, might need updating on Gentoo-side then, thanks.